src/app/_translate/rename-target-type.pipe.ts

```diff
@@ -0,0 +1,22 @@
+/** RenameTargetType
+/*
+/* a translation utility for html files
+/* supplies human-readable labels for target types
+*/
+import { Pipe, PipeTransform } from '@angular/core';
+
+const targetTypeNames = {
+  total: 'All',
```

Comment:
`All` of what?

Reply:
The context a target type - it's a label for all records of that type.

This has to fit into a very small area of UI real estate and has to be succinct.